frontend/src/components/object/ObjectPermission.vue

```diff
@@ -66,6 +66,7 @@ onBeforeMount(() => {
         class="ml-4"
         :bucket-id="object.bucketId"
         :object-id="object.id"
+        :object-name="object.name"
         :object-public="object.public"
```

Comment:
i guess we could just pass the whole `object` in one binding (`:object="object"`) instead of 4 separate ones, and break it down in the child component, and i think we've found some other ways since adopting this pattern early on.. but this is fine for now.